### PR TITLE
chore(operator): add openebs operator 2.6.0

### PR DESCRIPTION
**Pull request by akhilerm**
- add cstor operator 2.6.0
- add ndm operator 1.2.0
- add openebs operator 2.6.0
- add zfs operator 1.4.0
- add openebs-operator lite 2.6.0

Signed-off-by: Akhil Mohan <akhil.mohan@mayadata.io>

### CODE REVIEW

**Review by shubham14bajpai — APPROVED** [review]

**Review by prateekpandey14 — COMMENTED** [review]
looks good

**Comment by pawanpraka1** [comment]
we need to update the zfs operator yaml, there is a new value added for zfsvolume object (Failed).

**Review by pawanpraka1 — APPROVED** [review]
looks good.

File: 2.6.0/cstor-operator-RC2.yaml [hunk 1]
```diff
@@ -0,0 +1,5065 @@
...
+            - name: OPENEBS_CSI_API_URL
+              value: https://openebs.io
+              # OpenEBS namespace where the openebs cstor operator components
+              # has been installed
+            - name: OPENEBS_NAMESPACE
+              value: openebs
+              # Enable/Disable auto-remount feature, when volumes
+              # recovers form the read-only state
+            - name: REMOUNT
+              value: "false"
```
Comment (prateekpandey14) [inline 2.6.0/cstor-operator-RC2.yaml:1375]: change it to `true`

File: 2.6.0/cstor-operator-RC1.yaml [hunk 2]
```diff
@@ -0,0 +1,5065 @@
...
+            - name: OPENEBS_CSI_API_URL
+              value: https://openebs.io
+              # OpenEBS namespace where the openebs cstor operator components
+              # has been installed
+            - name: OPENEBS_NAMESPACE
+              value: openebs
+              # Enable/Disable auto-remount feature, when volumes
+              # recovers form the read-only state
+            - name: REMOUNT
+              value: "false"
```
Comment (prateekpandey14) [inline 2.6.0/cstor-operator-RC1.yaml:1375]: change it to `true`

File: 2.6.0/cstor-operator-RC2.yaml (hunk 1 above)
Comment (akhilerm) [reply 2.6.0/cstor-operator-RC2.yaml]: Done.

File: 2.6.0/cstor-operator-RC1.yaml (hunk 2 above)
Comment (akhilerm) [reply 2.6.0/cstor-operator-RC1.yaml]: Done.

**Review by shubham14bajpai — APPROVED** [review]

**Review by pawanpraka1 — APPROVED** [review]
zfs operator yaml is fine.

**Review by prateekpandey14 — APPROVED** [review]
lgtm

**Pull request merged**